core/src/test/java/nl/nn/adapterframework/xslt/XsltErrorTestBase.java

```diff
@@ -35,7 +35,7 @@
 	protected TestAppender testAppender;
 	private ErrorOutputStream errorOutputStream;
 	private PrintStream prevStdErr;
-	public static int EXPECTED_CONFIG_WARNINGS_FOR_XSLT2_SETTING=1;
+	public static int EXPECTED_CONFIG_WARNINGS_FOR_XSLT2_SETTING=0;
```

Comment:
Is er geen warning meer als er xslt2=true gezet wordt?
Idee hier is ook dat als de test verandert wordt van gebruik van 'xst2=true' naar 'xsltVersion=2', dat je dan deze variabele van 1 op 0 kan zetten.

Reply:
Dit wordt nu op een andere manier weergegeven, en komt dus niet meer in de test voor.